src/main/webapp/app/core/about-us/artemis-about-us.module.ts

```diff
@@ -0,0 +1,11 @@
+import { NgModule } from '@angular/core';
+import { AboutUsComponent } from 'app/core/about-us/about-us.component';
+import { AboutUsRoutingModule } from 'app/core/about-us/about-us-routing.module';
+import { TranslateModule } from '@ngx-translate/core';
+import { CommonModule } from '@angular/common';
+
+@NgModule({
+    declarations: [AboutUsComponent],
+    imports: [AboutUsRoutingModule, TranslateModule, CommonModule],
+})
+export class ArtemisAboutUsModule {}
```

Comment:
Why do we need two modules? One should be enough? (And only if we end up lazy loading, otherwise this should not be needed at all for a single component.

Reply:
Fixed